packages/sdk/src/assets/open-edition-nft.ts

```diff
@@ -1,20 +1,73 @@
-import {Hex, pad, toHex} from 'viem'
+import {Address, Hex, pad, toHex} from 'viem'
 
 import {GAS_CONTROLLER} from '../constants/gas-controllers'
 import {HolographOpenEditionERC721Contract} from '../contracts'
 import {queryTokenIdFromReceipt} from '../utils/decoders'
 import {HolographVersion, MintConfig, WriteContractOptions} from '../utils/types'
-import {NFT} from './nft'
-import {CreateNFT} from './nft.validation'
+import {CreateOpenEditionNFT, HolographOpenEditionNFTMetadata, validate} from './nft.validation'
+import {RequireMintedToken} from '../utils/decorators'
+import {getParsedTokenId} from '../utils/transformers'
+import {
+  HolographOpenEditionERC721ContractV1,
+  HolographOpenEditionERC721ContractV2,
+} from './holograph-open-edition-erc721-contract'
+import {NotMintedNFTError} from '../errors'
+
+export class OpenEditionNFT {
```

Comment:
We are not extending the NFT class anymore, why this change?

Reply:
This is due to the way metadata works. For standard NFTs, we can set individual metadata, but this isn't the case for Open Edition (OE) NFTs. Besides that, some metadata properties differ between standard NFTs and OE NFTs.